learn/what_is_meilisearch/language.md

```diff
@@ -1,42 +1,43 @@
 # Language
 
-**Meilisearch is multilingual**, featuring optimized support for:
+Meilisearch is multilingual, featuring optimized support for:
 
-- **Any language that uses whitespace to separate words**
-- **Chinese** (through [Jieba](https://github.com/messense/jieba-rs))
-- **Japanese** (through [Lindera](https://github.com/lindera-morphology/lindera))
+- Any language that uses whitespace to separate words
+- Chinese
+- Japanese
+- Hebrew
 
-We aim to provide global language support, and your feedback helps us [move closer to that goal](#improving-our-language-support). If you notice inconsistencies in your search results or the way your documents are processed, please open an issue on our [GitHub repository](https://github.com/meilisearch/meilisearch/issues/new/choose).
+We aim to provide global language support, and your feedback helps us move closer to that goal. If you notice inconsistencies in your search results or the way your documents are processed, please [open an issue in our tokenizer repo](https://github.com/meilisearch/charabia/issues/new).
 
-If you'd like to learn more about how different languages are processed in Meilisearch, see our [tokenizer documentation](/learn/advanced/tokenization.md).
+[Read more about our tokenizer](/learn/advanced/tokenization.md)
 
 ## Improving our language support
 
-While we have employees from all over the world at Meilisearch, we don't speak every language. In fact, we rely almost entirely on feedback from external contributors to know how our engine is performing across different languages.
+While we have employees from all over the world at Meilisearch, we don't speak every language. We rely almost entirely on feedback from external contributors to understand how our engine is performing across different languages.
 
-If you'd like to help us create a more global Meilisearch, please consider sharing your tests, results, and general feedback with us through [GitHub issues](https://github.com/meilisearch/Meilisearch/issues). Here are some of the languages that have been requested by users and their corresponding issue:
+If you'd like to request optimized support for a language that we don't currently support, please upvote the related [discussion in our product repository](https://github.com/meilisearch/product/discussions?discussions_q=label%3Aproduct%3Acore%3Atokenizer) or [open a new one](https://github.com/meilisearch/product/discussions/new) if it doesn't exist.
```

Comment:
There is a small mistake in my last proposal, here we create an issue in the good category.

```suggestion
If you'd like to request optimized support for a language that we don't currently support, please upvote the related [discussion in our product repository](https://github.com/meilisearch/product/discussions?discussions_q=label%3Aproduct%3Acore%3Atokenizer) or [open a new one](https://github.com/meilisearch/product/discussions/new?category=feedback-feature-proposal) if it doesn't exist.
```